### PR TITLE
Add GeneratePathProperty to xsd

### PR DESCRIPTION
**Pull request by KirillOsenkov**

### CODE REVIEW

**Review by Therzok — APPROVED** [review]

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd [hunk 1]
```diff
@@ -207,6 +207,13 @@ elementFormDefault="qualified">
                   </xs:documentation>
                 </xs:annotation>
               </xs:element>
+              <xs:element name="GeneratePathProperty">
+                <xs:annotation>
+                  <xs:documentation>
+                    <!-- _locID_text="PackageReference_GeneratePathProperty" _locComment="" -->Set to true to generate a Pkg* property that points to the restored location of the NuGet package contents
```
Comment (Forgind) [inline src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:213]: default?

**Review by Forgind — APPROVED** [review]

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (KirillOsenkov) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:213]: I was hoping someone more knowledgeable than me could push directly to this branch ;) 

Did you just fall into my trap?

Comment (Forgind) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd:213]:
```suggestion
                    <!-- _locID_text="PackageReference_GeneratePathProperty" _locComment="" -->Set to true to generate a Pkg* property that points to the restored location of the NuGet package contents. If unset, does not create the property.
```

**Review by Forgind — APPROVED** [review]

File: src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd (hunk 1 above)
Comment (Forgind) [reply src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd]: Maybe, but I do one-line changes from the browser to maximally ping you in the process 🙂

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**